node/_fs/_fs_readFile_test.ts

```diff
@@ -58,7 +58,7 @@ Deno.test("readFileEncodeBase64Success", async function () {
     });
   });
   assertEquals(typeof data, "string");
-  assertEquals(data as string, "aGVsbG8gd29ybGQ=");
+  assertEquals(data as string, "aGVsbG8gd29ybGQA");
```

Comment:
This is incorrect, Node returns the previous value. I'm not sure what changed between CLI versions though, looks like something to do with `Uint8Array::subarray()`.

Reply:
decode `aGVsbG8gd29ybGQA` results in `hello world` as well... this has been failing in canary for quite a while without anyone addressing it... 😢

Reply:
@kitsonk This is the fix: https://github.com/denoland/deno_std/pull/1279/commits/c0d7da82c48cceddf16cd631818175e298f957ab, the fixture changes there align with Node. It was an std issue probably exposed by some V8 patch.

Reply:
I am incorporating into this PR.